src/components/GatheringCard.tsx

```diff
@@ -0,0 +1,56 @@
+type TGathering = {
+  categoryList: string[]
+  title: string
+  date: string
+}
+
+type GatheringCardProps = {
+  data: TGathering
+}
+
+const GatheringCard = ({ data }: GatheringCardProps) => {
+  return (
+    <div className="card bg-slate-400 rounded-xl min-w-[190px] min-h-[185px] inline-flex flex-shrink-0 mr-3">
+      <div className="card-body bg-cardBg p-4 px-3 text-white relative">
+        <div className="flex items-center justify-between">
+          <div className="flex">
+            {data.categoryList.map(category => {
+              return (
+                <div key={category}>
+                  <div className="badge badge-neutral text-2xs mr-2">{category}</div>
+                </div>
```

Comment:
```suggestion
              <div key={category} className="badge badge-neutral text-2xs mr-2">{category}</div>
```

요거 dIv 로 감싸줘야 제대로 표시되나요?

Reply:
옷 ㅎㅎㅎ생략 가능하겠네요 :) 발견 감사합니다!